eng/config/PublishData.json

```diff
@@ -202,8 +202,8 @@
         "NonShipping"
       ],
       "vsBranch": "main",
-      "insertionCreateDraftPR": false,
-      "insertionTitlePrefix": "[dNext]"
+      "insertionCreateDraftPR": true,
+      "insertionTitlePrefix": "[Validation]"
```

Comment:
main is now the primary branch that we insert into VS main (equivalent to main-vs-deps before). I am not sure if its ok for the official insertion PRs to have a `[Validation]` prefix?